### DISCUSSION TITLE
Make your membership publicly visible

### DISCUSSION THREAD

**Discussion by oood** [abbrcode/abbreviations-in-code#92] [Ideas]
Hi guys, I just saw our members list is not visible when browsing our org page on a new device.

This is due to the fact that GitHub hides its organization's member list by default unless they choose to make it public.

I chose to make it public, you can make changes [here](https://github.com/orgs/abbrcodes/people) if you wish.

Once public, the organization will be listed on your GitHub user page.

It's okay if you don't want to make it public, just a suggestion.

Because at the moment I'm public and if no one goes public it makes me look like some kind of spokesperson or dictator and I respect your wishes and if none of you are going to go public I won't go any further.

 Use private browsing to view this page: https://github.com/abbrcodes

**Comment by oood** [comment]
@kisvegabor @philipeachille @T1xx1

**Reply by T1xx1** [reply]
I'm doing it this evening. I'm not at home rn.

**Reply by oood** [reply]
no problem!

**Reply by T1xx1** [reply]
Done

**Comment by T1xx1** [comment]
Only @kisvegabor missing

**Reply by kisvegabor** [reply]
Updated :)

**Comment by T1xx1** [comment]
Great
